ingestion/src/metadata/data_quality/interface/sqlalchemy/sqa_test_suite_interface.py

```diff
@@ -49,23 +49,25 @@ def __init__(
         self,
         service_connection_config: DatabaseConnection,
         ometa_client: OpenMetadata,
+        sampler: SamplerInterface,
         table_entity: Table = None,
-        sqa_metadata=None,
+        orm_table=None,
     ):
-        super().__init__()
-        self.ometa_client = ometa_client
-        self.table_entity = table_entity
-        self.service_connection_config = service_connection_config
+        super().__init__(
+            service_connection_config,
+            ometa_client,
+            sampler,
+            table_entity,
+        )
         self.create_session()
-        self._table = self._convert_table_to_orm_object(sqa_metadata)
```

Comment:
Have we kept the logic to pass the Metadata object when creating orm object. We need it to prevent naming conflict when creating the orm object if I well remember